Wrong link target for next step link.

tvuylsteke opened the pull request:
Corrected the link at the bottom. It pointed to an article about analyzing a packet trace while the wording suggested it would linke to the Elastic article to visualize the NSG log data.

azurecla commented:
Hi __@harelbr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tvuylsteke committed:
Update network-watcher-visualize-nsg-flow-logs-power-bi.md

Corrected the link at the bottom. It pointed to an article about analyzing a packet trace while the wording suggested it would linke to the Elastic article to visualize the NSG log data.
